### PR TITLE
chore: hide timeline bars and stack from ui mode

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: packages/trace-viewer/src/ui/timeline.tsx [hunk 1]
```diff
@@ -50,6 +51,10 @@ export const Timeline: React.FunctionComponent<{
 
   const { boundaries, offsets } = React.useMemo(() => {
     const boundaries = { minimum: model?.startTime || 0, maximum: model?.endTime || 30000 };
+    if (boundaries.minimum > boundaries.maximum) {
```
Comment (dgozman) [inline packages/trace-viewer/src/ui/timeline.tsx:54]: How is this possible?

File: packages/trace-viewer/src/ui/timeline.tsx [hunk 2]
```diff
@@ -95,41 +100,34 @@ export const Timeline: React.FunctionComponent<{
...
       const xMiddle = (bar.leftTime + bar.rightTime) / 2;
       const xd = Math.abs(time - xMiddle);
-      const yd = Math.abs(y - yMiddle);
       if (left > right)
         continue;
       // Prefer closest yDistance (the same bar), among those prefer the closest xDistance.
-      if (index === undefined ||
-          (yd < yDistance!) ||
-          (Math.abs(yd - yDistance!) < 1e-2 && xd < xDistance!)) {
+      if (index === undefined || xd < xDistance!) {
```
Comment (dgozman) [inline packages/trace-viewer/src/ui/timeline.tsx:118]: - Why this change?
- The comment does not match anymore.

Comment (pavelfeldman) [reply packages/trace-viewer/src/ui/timeline.tsx:118]: I don't think users could use it / were using it.

File: packages/trace-viewer/src/ui/timeline.tsx (hunk 1 above)
Comment (pavelfeldman) [reply packages/trace-viewer/src/ui/timeline.tsx:54]: Initial values min is maxint, max is minint.
